src/Service/UploadFromOpenCFP/UploadedCsvCreateSpeakerTalk.php

```diff
@@ -24,26 +25,42 @@ final class UploadedCsvCreateSpeakerTalk
     private const SPEAKER_EMAIL = 'email';
 
     private $talkTitleColumnNumber;
+
     private $talkDescriptionColumnNumber;
+
     private $emailExportTalkTitleColumnNumber;
+
     private $speakerFirstNameColumnNumber;
+
     private $speakerLastNameColumnNumber;
+
     private $speakerEmailColumnNumber;
+
     private $speakerRepository;
+
     private $talkRepository;
+
     /**
      * @var Speaker[]
      */
     private $addedSpeakersList = [];
+
     /**
      * @var Talk[]
      */
     private $addedTalksList = [];
 
-    public function __construct(SpeakerRepositoryInterface $speakerRepository, TalkRepositoryInterface $talkRepository)
-    {
+    /** @var TalkFactory */
```

Comment:
This type can be inferred from the constructor type hint, please remove it. (less comment maintenance is better).

Reply:
Constructor type inference is not always well supported and is definitely harder to read for a human (you have to find the constructor's code to find out the type of properties), so I don't really agree on this one... but this won't be a problem when we finally get PHP7.4.
Meanwhile, I'll remove the comments